src/main/java/org/radarcns/util/commandline/CommandLineArgs.java

```diff
@@ -0,0 +1,28 @@
+package org.radarcns.util.commandline;
+
+import com.beust.jcommander.Parameter;
+
+public class CommandLineArgs {
+
+    @Parameter(names = { "-f", "--format" }, description = "Format to use when converting the files. JSON and CSV is available.")
+    public String format = "csv";
+
+    @Parameter(names = { "-c", "--compression" }, description = "Compression to use when converting the files. Gzip is available.")
+    public String compression = "none";
+
+    // Default set to false because causes loss of records from Biovotion data. https://github.com/RADAR-base/Restructure-HDFS-topic/issues/16
+    @Parameter(names = { "-d", "--deduplicate" }, description = "Boolean to define if to use deduplication or not.", validateWith = BooleanValidator.class)
+    public Boolean deduplicate = false;
```

Comment:
jcommander supports on/off flags, right? Boolean should not be needed, just the `-d` flag. Like `--help`